Blockzilla/BrowserViewController.swift

```diff
@@ -406,6 +406,10 @@ class BrowserViewController: UIViewController {
         urlBar.dismissTextField()
     }
     
+    public func deactivateUrlBar() {
+        urlBar.dismiss()
```

Comment:
Is an extra method really necessary?

Reply:
To do just this thing, yes. Do you have another idea?

Reply:
I noticed another method `deactivateUrlBarOnHomeView()` and I was wondering if it does the same thing.

Reply:
I saw that too, but it does something else, not working in this case.